dataikuapi/dss/admin.py

```diff
@@ -365,6 +378,79 @@ def save(self):
         self.client._perform_empty("PUT", "/current-user", body = self.settings)
 
 
+class DSSUserActivity(object):
+    """
+    Settings for a DSS user.
+    Do not call this directly, use :meth:`DSSUser.get_activity` or :meth:`DSSClient.list_users_activity`
+    """
+
+    def __init__(self, client, login, activity):
+        self.client = client
+        self.login = login
+        self.activity = activity
+
+    def get_raw(self):
+        """
+        Get the raw activity of the user as a dict.
+
+        :return: the raw activity
+        :rtype: dict
+        """
+        return self.activity
+
+    def get_last_successful_login_attempt(self, as_date=False):
```

Comment:
I would not specially say `attempt` in this situation and probably in all the functions in this file